tmt/base.py

```diff
@@ -3094,10 +3094,11 @@ def _resolve_import_from_fmf_cache(self, reference: RemotePlanReference) -> Iter
 
                 reference.resolve_dynamic_ref(tmpdirname, self)
 
+        assert reference.url
```

Comment:
Please, move the assert at the beginning of the method - see `_resolve_import_from_git()` and line 3040, it's the exact same case, narrowing the type we know is no longer `None` but linter can't spot it.

Reply:
done - df49d75e